src/bindings/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ if(LIBCELLML_BINDINGS_PYTHON) # Or any other bindings that use SWIG.
 endif()
 
 if(LIBCELLML_BINDINGS_PYTHON)
+  if(NOT LIBCELLML_BUILD_SHARED)
+    message(WARNING "The Python bindings are not intended to work with static libraries.")
```

Comment:
Might want to explain why since we can build and use the Python bindings on Windows static?

Reply:
We can do so on Linux and macOS as well, it just doesn't make sense given that we create multiple modules. I considered writing more of an explanation but decided the warning would get too long.

Reply:
The current warning is exactly the kind of warning I don't like. It is effectively me telling not to do something that... it allows me to do. Just doesn't make sense to me. (FWIW, libOpenCOR only allows to build Python bindings in static mode.)

Reply:
I agree, if its not intended to work then this should be an error - and the message definitely reads like an error. But I suspect its just the wording of the message rather than an actual issue. Do we use static for the builds on Windows? Should we make this message platform specific?

I think at a minimum it should reworded as a warning message like "The Python bindings are not recommended to be built with static libraries".

Reply:
Still not super happy with that warning message, but... the original issue is addressed, so... meh.